lib/compute/agent-nodes.ts

```diff
@@ -35,7 +35,8 @@ export class AgentNode {
 
   public readonly InitScript: string = 'sudo mkdir -p /var/jenkins/ && sudo chown -R ec2-user:ec2-user /var/jenkins '
     + '&& sudo yum install -y java-1.8.0-openjdk cmake python3 python3-pip && sudo yum groupinstall -y \'Development Tools\' '
-    + '&& sudo ln -sfn `which pip3` /usr/bin/pip && pip3 install pipenv && sudo ln -s ~/.local/bin/pipenv /usr/local/bin';
+    + '&& sudo ln -sfn `which pip3` /usr/bin/pip && pip3 install pipenv && sudo ln -s ~/.local/bin/pipenv /usr/local/bin '
```

Comment:
Would it make sense to move this init script at config level so that it is separate for each agent node?
We started with common script because both were AL2 agent nodes but Ubuntu, windows, etc will be different. Can be part of another PR though.

Reply:
We havent think about windows for now, but only using AL2 for LINUX Agent is enough, as AL2 can provision Ubuntu container for buildx purposes.

Reply:
`AL2 can provision Ubuntu container for buildx purposes` I didn't get that part? You mean we can pull ubuntu image on AL2?

Reply:
I tried that and it works

Reply:
I was able to pull the ubuntu docker image on AL2X64 and run the job